src/main/java/io/github/llamarama/team/common/entity/caravantrader/CaravanTraderEntity.java

```diff
@@ -29,6 +29,7 @@
 public class CaravanTraderEntity extends MerchantEntity {
 
     private boolean hasLlama;
+    private int lifespan = 36000;
```

Comment:
This should be saved to the Nbt of the entity so that it still keeps ticking after a server restart and doesn't start over.